src/cookies.rs

```diff
@@ -1,25 +1,109 @@
 //! Cookie-related functionality for WebDriver.
 use serde::{Deserialize, Serialize};
 use time::OffsetDateTime;
-use webdriver::command::WebDriverCommand;
+use webdriver::command::{AddCookieParameters, WebDriverCommand};
+use webdriver::common::Date;
 
 use crate::client::Client;
 use crate::error;
 
 /// Type alias for a [cookie::Cookie]
 pub type Cookie<'a> = cookie::Cookie<'a>;
 
+/// Wrapper for serializing AddCookieParameters.
+#[derive(Debug, Serialize)]
+pub struct AddCookieParametersWrapper<'a> {
+    /// The cookie to serialize.
+    #[serde(with = "AddCookieParameters")]
+    pub cookie: &'a AddCookieParameters,
```

Comment:
This would make `webdriver` a public dependency again, so we can't have this.

Reply:
Ah, yep I missed the fact that traits are public.